docs/heasarc/heasarc.rst

```diff
@@ -11,292 +11,272 @@ This is a python interface for querying the
-    RH701576N00          HRI       68154 3C 273               ... 2.0500 49704.3090856482 49724.6236226852  0.190 (3c273)\n
-    RP600242A01          PSPCB     24822 GIOVANELLI-HAYNES CL ... 1.6000 48980.6468865741 48982.9284837963 34.236 (3c273)\n
-    RH700234N00          HRI       17230 3C 273               ... 2.0500 48629.2693055556 48632.4716782407  0.190 (3c273)\n
-
-Alternatively, a query can also be conducted around a specific set of sky
-coordinates:
+Query a Table
+-------------
+The basic use case is one where we wants to query a table from some position in the sky.
+In this example, we query the NuSTAR master table ``numaster`` for all observations
+of the AGN ``NGC 3783``. We use `~astropy.coordinates.SkyCoord` to obtain the coordinates
+and then pass them to `~astroquery.heasarc.HeasarcClass.query_region`:
 
 .. doctest-remote-data::
 
     >>> from astroquery.heasarc import Heasarc
     >>> from astropy.coordinates import SkyCoord
-    >>> heasarc = Heasarc()
-    >>> mission = 'rosmaster'
-    >>> coords = SkyCoord('12h29m06.70s +02d03m08.7s', frame='icrs')
-    >>> table = heasarc.query_region(coords, mission=mission, radius='1 degree')
-    >>> table[:3].pprint(max_width=120)
-         SEQ_ID      INSTRUMENT EXPOSURE ...    START_TIME        END_TIME                 SEARCH_OFFSET_            
-                                   s     ...       mjd              mjd                                              
-    ---------------- ---------- -------- ... ---------------- ---------------- --------------------------------------
-    RH701576N00          HRI       68154 ... 49704.3090856482 49724.6236226852  0.191 (187.2779228198,2.0524148595)\n
-    RP600242A01          PSPCB     24822 ... 48980.6468865741 48982.9284837963 34.237 (187.2779228198,2.0524148595)\n
-    RH700234N00          HRI       17230 ... 48629.2693055556 48632.4716782407  0.191 (187.2779228198,2.0524148595)\n
-
-Note that the :meth:`~astroquery.heasarc.HeasarcClass.query_region` converts
-the passed coordinates to the FK5 reference frame before submitting the query.
-
-
-Modifying returned table columns
---------------------------------
-
-Each table has a set of default columns that are returned when querying the
-database. You can return all available columns for a given mission by specifying
-the ``fields`` parameter in either of the above queries. For exampe:
+    >>> pos = SkyCoord.from_name('ngc 3783')
+    >>> tab = Heasarc.query_region(pos, table='numaster')
+    >>> tab['name', 'obsid', 'ra', 'dec'][:3].pprint()
+      name    obsid      ra      dec   
+                        deg      deg   
+    -------- -------- -------- --------
+    NGC_3783 80701617 174.7573 -37.7386
+    NGC_3783 60901023 174.7571 -37.7385
+    NGC_3783 60902005 174.7571 -37.7385
+
+To query a region around some position, specifying the search radius,
+we use `~astropy.units`:
```

Comment:
don't refer to the module, but the classes.

Reply:
changed.